src/OpenTelemetry.Instrumentation.Http/HttpWebRequestInstrumentationOptions.netfx.cs

```diff
@@ -32,9 +32,13 @@ public class HttpWebRequestInstrumentationOptions
         public bool SetHttpFlavor { get; set; }
 
         /// <summary>
-        /// Gets or sets <see cref="ITextFormat"/> for context propagation. Default value: <see cref="TraceContextFormat"/>.
+        /// Gets or sets <see cref="ITextFormat"/> for context propagation. Default value: <see cref="CompositePropagator"/>.
         /// </summary>
-        public ITextFormat TextFormat { get; set; } = new TraceContextFormat();
+        public ITextFormat TextFormat { get; set; } = new CompositePropagator(new System.Collections.Generic.List<ITextFormat>
```

Comment:
Any reason not to put `using System.Collections.Generic` (guess there shouldn't be naming conflicts)?

Reply:
just updated